docs/strategies/rust.md

```diff
@@ -71,249 +75,3 @@ version, whether it is platform-specific, what build category it's included in
 (build, dev, or standard), and what features it compiles with.  For any further
 information, we have to compare this package and version with the corresponding
 entry in `.packages`.
-
-## Schema
```

Comment:
Why delete this?

Reply:
I don't think it provides much value to our users.  Especially since this is not any output that they would normally see.  Also, there is a link to the schema in the text already, which will be kept up-to-date if there are any changes, so no documentation rot.